kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -795,6 +795,45 @@ def foundry_section_edge(
     apr_proof.write_proof()
 
 
+def foundry_get_model(
+    foundry_root: Path,
+    test: str,
+    nodes: Iterable[NodeIdLike] = (),
+    pending: bool = False,
+    failing: bool = False,
+) -> str:
+    contract_name = test.split('.')[0]
+    foundry = Foundry(foundry_root)
+    proofs_dir = foundry.out / 'apr_proofs'
+
+    contract_name, test_name = test.split('.')
+    proof_digest = foundry.proof_digest(contract_name, test_name)
+    proof = Proof.read_proof(proof_digest, proofs_dir)
+    assert isinstance(proof, APRProof)
+
+    if not nodes:
+        _LOGGER.warning('Node ID is not provided. Displaying models of failing and pending nodes:')
+        failing = pending = True
+
+    if pending:
+        nodes = list(nodes) + [node.id for node in proof.pending]
+    if failing:
+        nodes = list(nodes) + [node.id for node in proof.failing]
+    nodes = unique(nodes)
+
+    res_lines = []
+
+    with KCFGExplore(foundry.kevm, id=proof.id) as kcfg_explore:
+        for node_id in nodes:
+            res_lines.append('')
+            res_lines.append(f'Node id: {node_id}')
+            node = proof.kcfg.node(node_id)
+            model_info = print_model(node, kcfg_explore)
+            res_lines.extend(model_info)
```

Comment:
```suggestion
            res_lines.extend(print_model(node, kcfg_explore))
```
I'm not sure which is better (same for other places this occurs).

Reply:
Thanks! I agree, done in https://github.com/runtimeverification/evm-semantics/commit/13c0f97d313913202f9a79bdd796b15915959bab.